ngraph/cmake/external_protobuf.cmake

```diff
@@ -14,13 +14,34 @@
 # limitations under the License.
 # ******************************************************************************
 
-# Enable ExternalProject CMake module
-include(ExternalProject)
+include(FetchContent)
 
 #------------------------------------------------------------------------------
 # Download and install Google Protobuf ...
 #------------------------------------------------------------------------------
 
+# Since this file is going to be modifying CMAKE_*_FLAGS we need to preserve
+# it so we won't overwrite the caller's CMAKE_*_FLAGS
+set(PUSH_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+set(PUSH_CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE}")
+set(PUSH_CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
+set(PUSH_CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE}")
+set(PUSH_CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE}")
+set(PUSH_CMAKE_MODULE_LINKER_FLAGS_RELEASE "${CMAKE_MODULE_LINKER_FLAGS_RELEASE}")
```

Comment:
why LTO does not work for onnx?

Reply:
Protobuf library have problems with pthread if LTO is enabled.